### PR TITLE
Spread out pod status updates to apiserver.

### PR DESCRIPTION
**Pull request by vmarmol**
Lowers pod status interval to every 2m and spreads updates within that
time window.

### CODE REVIEW

File: cmd/kubelet/app/server.go [hunk 1]
```diff
@@ -83,13 +84,14 @@ type KubeletServer struct {
...
 func NewKubeletServer() *KubeletServer {
 	return &KubeletServer{
-		SyncFrequency:         10 * time.Second,
-		FileCheckFrequency:    20 * time.Second,
-		HTTPCheckFrequency:    20 * time.Second,
-		StatusUpdateFrequency: 20 * time.Second,
-		EnableServer:          true,
-		Address:               util.IP(net.ParseIP("127.0.0.1")),
-		Port:                  ports.KubeletPort,
+		SyncFrequency:            10 * time.Second,
```
Comment (dchen1107) [inline cmd/kubelet/app/server.go]: What is the diff here? Same for FileCheckFrequency, and other below except PodStatusUpdateFrequency

File: cmd/kubelet/app/server.go [hunk 2]
```diff
@@ -273,21 +277,22 @@ func SimpleRunKubelet(client *client.Client,
...
-		MinimumGCAge:            10 * time.Second,
-		MaxPerPodContainerCount: 5,
-		MaxContainerCount:       100,
-		MasterServiceNamespace:  masterServiceNamespace,
-		VolumePlugins:           volumePlugins,
-		TLSOptions:              tlsOptions,
-		CadvisorInterface:       cadvisorInterface,
-		ConfigFile:              configFilePath,
-		ImageGCPolicy:           imageGCPolicy,
+		Port:                     port,
```
Comment (dchen1107) [inline cmd/kubelet/app/server.go]: ditto and below many lines

File: cmd/kubelet/app/server.go (hunk 1 above)
Comment (vmarmol) [inline cmd/kubelet/app/server.go]: It only adds an initialization of PodStatusUpdateFrequency. That made Gofmt redo the spacing which is why there is such a large diff.

**Comment by dchen1107** [comment]
Why not introducing bulk update on PodStatus? instead of introducing spreading out logic?

**Comment by vmarmol** [comment]
@dchen1107 this holds us up until then. It's meant to be a stop-gap since bulk is not v1. If we'd rather do that then SGTM.

**Comment by dchen1107** [comment]
LGTM

**Pull request merged**